src/client.h

```diff
@@ -51,6 +39,21 @@ class Client : public Object {
     int basew_, baseh_, incw_, inch_, maxw_, maxh_, minw_, minh_;
     // for other modules
     Signal_<HSTag*> needsRelayout;
+
+    // attributes:
+    Attribute_<bool> urgent_;
+    Attribute_<bool> fullscreen_;
+    Attribute_<std::string> title_;  // or also called window title; this is never NULL
+    Attribute_<std::string> window_id_str;
+
+    Attribute_<std::string> keyMask_; // keymask applied to mask out keybindins
+    Attribute_<bool> pseudotile_; // only move client but don't resize (if possible)
+    bool        ewmhrequests_ = true; // accept ewmh-requests for this client
```

Comment:
Why these spaces after `bool`?

Reply:
I've only reordered lines and this was from a previous alignment.

Reply:
Ah! I'd suggest cleaning that up while at it. It wouldn't make the diff any harder to read, imho.

Reply:
Ah, I've just noticed the "(keep)" in your question.
These lines will eventually be replaced by `Attribute_<bool>`, so let us please not worry about the spaces for the Moment.

Reply:
Alright, nevermind then.